tests/endpoints/bookstore_grpc/server/server.go

```diff
@@ -266,6 +267,22 @@ func (s *BookstoreServerV1Impl) CreateShelf(ctx context.Context, req *bspbv1.Cre
 		return nil, err
 	}
 
+	// Unmarshal, deepcopy and marshal, to verify the received binary `any`
+	if req.Shelf.Any != nil {
+		var book bspbv1.Book
+		err := ptypes.UnmarshalAny(req.Shelf.Any, &book)
```

Comment:
can we use a message type that is not used in any rpc call?  In OP config, this message has to be added into "types" field.  But in Envoy transcoding,  I hope the descriptor should include it. so it will work. But we need to test it out.

Reply:
ACK

Reply:
Done.